lib/constants/error-messages.ts

```diff
@@ -0,0 +1,44 @@
+// Platform Error
+export const PLATFORM_FAILURE = 'platform-failure';
+export const PLATFORM_NOT_FOUND = 'platform-not-found';
+export const BAD_CREDENTIALS = 'bad-credentials';
+export const INTEGRATION_UNAUTHORIZED = 'integration-unauthorized';
+export const RATE_LIMIT_EXCEEDED = 'rate-limit-exceeded';
+export const AUTHENTICATION_ERROR = 'authentication-error';
+export const AUTHENTICATION_FAILURE = 'authentication-failure';
+
+// System error
+export const INSUFFICIENT_DISK_SPACE = 'disk-space';
+
+// Repository Error
+export const REPOSITORY_CANNOT_FORK = 'cannot-fork';
+export const REPOSITORY_EMPTY = 'empty';
+export const REPOSITORY_TEMPORARY_ERROR = 'temporary-error';
+export const REPOSITORY_CHANGED = 'repository-changed';
+export const REPOSITORY_RENAMED = 'renamed';
+export const REPOSITORY_ARCHIVED = 'archived';
+export const REPOSITORY_DISABLED = 'disabled';
+export const REPOSITORY_NOT_FOUND = 'not-found';
+export const REPOSITORY_ACCESS_FORBIDDEN = 'forbidden';
+export const REPOSITORY_BLOCKED = 'blocked';
+export const REPOSITORY_MIRRORED = 'mirror';
+export const REPOSITORY_FORKED = 'fork';
+export const REPOSITORY_NO_VULNERABILITY = 'no-vulnerability-alerts';
+export const REPOSITORY_UNINITIATED = 'uninitiated';
+
+// Manager Error
+export const LOCKFILE_ERROR = 'lockfile-error';
+export const NO_PACKAGE_FILES = 'no-package-files';
+
+// Registry error
+export const REGISTRY_FAILURE = 'registry-failure';
+
+// Unknown Error
+export const UNKNOWN_ERROR = 'unknown-error';
+
+// File Error
+export const FILE_UPDATE_FAILED = 'update-failure';
+
+export const BUNDLER_INVALID_CREDENTIALS = 'bundler-credentials';
+export const BUNDLER_COULD_NOT_RESOLVE = 'bundler-resolve';
+export const BUNDLER_UNKNOWN_ERROR = 'bundler-unknown';
```

Comment:
If these three are "local" and only used within lib/manager/bundler/* then we don't need to define them here.

Reply:
Actually I just noticed we use these within branch worker: `} else if (err.message.startsWith('bundler-')) {`